test/dockerpy/images.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+import docker
+
+busybox = "docker.io/library/busybox:latest"
+
+def get_client():
+   return docker.DockerClient(base_url="unix://var/run/docker.sock")
```

Comment:
```suggestion
def get_client():
   return docker.Client(base_url="unix://var/run/docker.sock")
```
From https://docker-py.readthedocs.io/en/1.6.0/api/

This should be in a setup fixture